server/paths/refer.ts

```diff
@@ -62,5 +69,11 @@ export default {
   showPerson: referralPersonPath,
   start: startReferralPath,
   submit: submitReferralPath,
+  submitted: {
+    additionalInformation: submittedReferralAdditionalInformationPath,
+    personalDetails: submittedReferralPersonalDetailsPath,
+    programmeHistory: submittedReferralProgrammeHistoryPath,
+    sentenceInformation: submittedReferralSentenceInformationPath,
+  },
```

Comment:
I wonder if it would be better to nest the other way around, given that's how the paths work? So the other paths would be nested in `draft`, following the path patterns, rather than the submitted ones being nested?

Reply:
Yeah possibly, suspect it would be a big'ish find and replace PR so best to do separately if we think that's the right way to go.